From f27149432b090bc62802e26f3ca2b333fe4335d3 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 20 Oct 2017 16:27:41 -0700
Subject: [PATCH] IMGAPI-653 imgapi's sapi_template for ~/.mantaprofile gets
 MANTA_USER wrong Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
 Approved by: Todd Whiteman <todd.whiteman@joyent.com>

---
 sapi_manifests/mantaprofile/template | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sapi_manifests/mantaprofile/template b/sapi_manifests/mantaprofile/template
index bd274aa..a807b22 100644
--- a/sapi_manifests/mantaprofile/template
+++ b/sapi_manifests/mantaprofile/template
@@ -1,3 +1,3 @@
-export MANTA_USER={{{ufds_admin_login}}}
+export MANTA_USER={{{IMGAPI_MANTA_USER}}}
 export MANTA_URL={{{IMGAPI_MANTA_URL}}}
 export MANTA_KEY_ID={{{IMGAPI_KEY_ID}}}
-- 
2.21.0

